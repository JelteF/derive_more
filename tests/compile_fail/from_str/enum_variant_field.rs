#[derive(derive_more::FromStr)]
enum Enum {
    Unit,
    Tuple(i32),
}

fn main() {}
