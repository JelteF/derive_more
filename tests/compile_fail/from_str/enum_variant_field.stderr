error: only enum variants with no fields can derive `FromStr` unless `#[from_str(forward)]` is specified
 --> tests/compile_fail/from_str/enum_variant_field.rs:4:10
  |
4 |     Tuple(i32),
  |          ^^^^^
